etna/collections/models.py

```diff
@@ -543,15 +542,9 @@ class Highlight(Orderable):
         on_delete=models.SET_NULL,
         verbose_name=_("image"),
     )
-    long_description = RichTextField(
-        verbose_name=_("long description"),
-        features=settings.RESTRICTED_RICH_TEXT_FEATURES,
-        max_length=400,
-    )
 
     panels = [
         FieldPanel("image"),
-        FieldPanel("long_description"),
     ]
 
     def clean(self) -> None:
```

Comment:
Hey @jamesbiggs,

this works locally for me. I would suggest that we do something similar here in terms of checking that a record has an image, but we check that the record has a description. The design is dependent on it.